src/pages/jobs.md

```diff
@@ -9,7 +9,7 @@ description: Shorebird openings
 We're hiring
 
 - [Founding Engineer](#founding-engineer)
-- [Founding Product Manager](#founding-product-manager)
+- [Founding "Business Person"](#founding-business-person)
```

Comment:
I don't have a great suggestion for an alternative but I do feel that calling this role: "Business Person" will be a turn-off for a lot of folks because it sounds a bit unserious and hand wavy. I imagine it would be the equivalent of renaming our Founding Engineer to "Founding Coding Person" 🤷

Reply:
Fair.  I think "Product Manager" is wrong.  This is what ChatGPT said:
https://chatgpt.com/share/6760933f-4f78-8002-9af4-946bcfddc8ab

Reply:
Maybe just "Head of Growth" 🤷

Reply:
Renamed to "business generalist".